schema/pet-insurance.json

```diff
@@ -505,12 +487,8 @@
                             "example": "1976-01-27"
                         }
                     }
-                }
-            },
-
-            "Address": {
-                "type": "object",
-                "properties": {
+                },
+                "address": {

```

Comment:
We have `Generic.Address` — why don't we use it here?

Reply:
Because there are elements not needed (like: City, Country)

Reply:
Ok, this seems like a broader topic (fyi we had a discussion about addresses last week). Feel free to merge this one for now.

Reply:
@Xedros the domain must be as universal as possible. I'm pretty sure you'll find a pet insurance provider requiring a more complete address.
This domain must cater to as many future pet insurance providers as possible, not just the one you've scripted.